Formula/haskell-language-server.rb

```diff
@@ -0,0 +1,28 @@
+class HaskellLanguageServer < Formula
+  desc "Integration point for ghcide and haskell-ide-engine. One IDE to rule them all"
+  homepage "https://github.com/haskell/haskell-language-server"
+  url "https://github.com/haskell/haskell-language-server/archive/1.0.0.tar.gz"
+  sha256 "14e28d6621d029f027fae44bc4a4ef62c869dab24ff01b88a2e51e6679cbff6c"
+  license "Apache-2.0"
+  head "https://github.com/haskell/haskell-language-server.git"
+
+  livecheck do
```

Comment:
Can we add a comment before this line that says that we need `:github_latest` here because otherwise `livecheck` picks up spurious non-release tags? It seems some users are confused about when to use `:github_latest`, so we don't want to mislead them when they see this.

Reply:
Added